src/praisonai-agents/praisonaiagents/runtime/health_registry.py

```
@@ -0,0 +1,151 @@
+    def run(
+        self,
+        context: Mapping[str, Any],
+        *,
+        fix: bool = False,
+    ) -> List[HealthCheckResult]:
+        results: List[HealthCheckResult] = []
+        self.get_checks()
+        for check_id, check in self._checks.items():
+            try:
+                findings = list(check.detect(context))
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)